packages/cfpb-icons/src/icons-svg-inline.js

```diff
@@ -1,4 +1,4 @@
-const path = require('path');
+import path from 'path';
```

Comment:
@anselmbradford this change seems to have broken the GitHub Actions check that builds the docs and runs Lighthouse against them, see [here](https://github.com/cfpb/design-system/actions/runs/3643893956/jobs/6152560324):

> Cannot use import statement outside a module
>      Error in /home/runner/work/design-system/design-system/node_modules/@cfpb/cfpb-icons/src/icons-svg-inline.js (line null, column -1)

The command being run is `yarn build-netlify`. Would you mind taking a look at this failure?

Reply:
Spun out into https://github.com/cfpb/design-system/pull/1510 for easier review. Thanks!